### ISSUE TITLE
Fix invalid PreTeXt code

### ISSUE THREAD

**Issue by mitchkeller** [OpenDiscreteMath/ibl-combinatorics#9]
We need to make sure that all the files are valid PTX so that we can compile the whole book before we start matching up the compiled version with Bogart's final PDF.

**Comment by oscarlevin** [comment]
Except for the xrefs, this is done.  I started a new issue for xrefs.
